data/bedrock/latest/proto.yml

```diff
@@ -3878,3 +3892,31 @@ packet_camera_instruction:
    !bound:
    # Data is a compound tag of the instructions to sent.
    data: nbt
+
+packet_compressed_biome_definitions:
+   !id: 0x12d
+   !bound: client
+   # via PMMP:
+   # This packet is only sent by the server when client-side chunk generation is enabled in vanilla. It contains NBT data
+   # for biomes, similar to the BiomeDefinitionListPacket, but with a large amount of extra data for client-side chunk
+   # generation.
+   # 
+   # The data is compressed with a cursed home-brewed compression format, and it's a miracle it even works.
+   raw_payload: string
```

Comment:
TODO : actually write out the structure for this , via https://github.com/Sandertv/gophertunnel/pull/184/files#diff-b20e2824b5fcdd69916f6958cbe85bf528a2a66c93feec81c751be19b546cb5aR433